docs/content/box.md

```diff
@@ -31,13 +32,79 @@ Type of storage to configure.
 Choose a number from below, or type in your own value
 [snip]
 XX / Box
+ 1 / 1Fichier
+   \ "fichier"
+ 2 / A stackable unification remote, which can appear to merge the contents of several remotes
+   \ "union"
```

Comment:
This still seems to have all the other cloud providers?

This is what it should look like

```
Type of storage to configure.
Choose a number from below, or type in your own value
[snip]
XX / Box
   \ "box"
[snip]
Storage> box
```